west.yml

```diff
@@ -101,7 +101,11 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 176e8083b059dbbb387d4bd540c4b3f32b27b6cd
+
+
+      revision: pull/287/head
```

Comment:
Please point to 8e6bf748d6e61bcbcf7161d8d60910a1ec0eb131

Reply:
Done and rebased as well